src/common/api.js

```diff
@@ -24,8 +24,11 @@ export function callApi(path) {
     });
 }
 
```

Comment:
@vinhkq `dispatch` should be `action`.
and then `if(dispatch.id)` should be `if(action and action.id)`